packages/wallet/src/services/TransactionsTracker.ts

```diff
@@ -382,6 +398,56 @@ export const createTransactionsTracker = (
     )
   );
 
+  const signed$ = new TrackerSubject<SignedTx[]>(
+    merge(
+      newSignedOrPreviouslySigned$.pipe(
+        mergeMap((signedTx) => signedTx),
```

Comment:
Just a naming convention for observables
```suggestion
        mergeMap((signedTx$) => signedTx$),
```

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/1113/commits/3124f542730a07f0199082b29c004747f300d0ef